config/environments/production.rb

```diff
@@ -42,7 +40,10 @@
      :s3_credentials => {
          :bucket => ENV['AWS_BUCKET'],
          :access_key_id => ENV['AWS_ACCESS_KEY_ID'],
-         :secret_access_key => ENV['AWS_SECRET_ACCESS_KEY']
+         :secret_access_key => ENV['AWS_SECRET_ACCESS_KEY'],
```

Comment:
Use the new Ruby 1.9 hash syntax.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.